statistics/update.go

```diff
@@ -340,6 +340,9 @@ func (h *Handle) dumpTableStatColSizeToKV(id int64, delta variable.TableDelta) e
 		}
 		values = append(values, fmt.Sprintf("(%d, 0, %d, 0, %d)", id, histID, deltaColSize))
 	}
+	if len(values) == 0 {
```

Comment:
Why change it?

Reply:
Something is wrong with github? I think I have answer this.

Reply:
https://github.com/pingcap/tidb/pull/7147/files/2df5d1d246cac859f3836b7a553bb9f13bcacd7d#diff-43d226f85e4ecd4c612f6b737dd3ef61

Reply:
Can not see your answer. Please post it again.

Reply:
> coocood 2 days ago  Member
> Why change this?
> 
>   @tiancaiamao
> tiancaiamao 2 days ago  Member
> Statistics for table partition is not done yet, so it will meet some error during constructing the SQL.
> The problem is, as long as any error is met, the DumpStatsDeltaToKV process would stop, then > TestShowStatsHealthy will fail.
> 
> That error is irrelevant to the whole statistics functional, we should ignore that error and continue, that's why I change here.
> 
> This change should be reasonable. @lamxTyler
> 
>   @lamxTyler
> lamxTyler a day ago  Member
> Yes, it is reasonable.